sdk/communication/AzureCommunicationChat/Source/ChatClient.swift

```diff
@@ -65,6 +68,21 @@ public class ChatClient {
         )
 
         self.service = client.chat
+
+        var token: String?
+
+        credential.token(completionHandler: {(communicationAccessToken, _)
+            in
+            if communicationAccessToken != nil {
+                token = communicationAccessToken!.token
+            }
```

Comment:
Use `if let` instead of force unwrap. Swiftlint should consider this an error.

Reply:
changed to if let instead of force unwrap